README.md

```diff
@@ -1,3 +1,189 @@
+
+    Example code at time of writing shown, please use link if this is not recent. 
+
+    ```
+    # Remove older versions of docker. It's fine if apt-get reports that none of the packages are installed
+    sudo apt-get remove docker docker-engine docker.io containerd runc
+    # Update apt
+    sudo apt-get update
+    # Install packages to allow apt to use repository over HTTPS
+    sudo apt-get install \
+        apt-transport-https \
+        ca-certificates \
+        curl \
+    software-properties-common
+    # Add Docker's official GPG key
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+    # Verify key by searching for the last 8 characters of fingerprint
+    sudo apt-key fingerprint 0EBFCD88
+    # Add stable repository for amd64 architecture
+    sudo add-apt-repository \
+        "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+        $(lsb_release -cs) \
+        stable"
+    # Update apt
+    sudo apt-get update
+    # Install Docker ce
+    sudo apt-get install docker-ce
+    ```
+
+### Installation choice 1: Python's virtual enviroment library, [venv](https://docs.python.org/3/library/venv.html)
+
+1. Clone repository 
+
+    ```
+    git clone https://github.com/stefpiatek/cnv-patissier.git
+    ```
+
+2. With **Python 3.6**, set up virtual environment
+    
+    ```
+    cd cnv-patissier
+    # Create the virtual environment
+    python3.6 -m venv .venv 
+    # Enable the virtual environment
+    source .venv/bin/activate
+    ```
+
+3. Install requirements
+
+    ```
+    pip install -r requirements.txt
+    ```
+
+### Installation choice 2: [Anaconda](https://conda.io/en/latest/index.html)
+
+1. Clone repository 
+
+    ```
+    git clone https://github.com/stefpiatek/cnv-patissier.git
+    ```
+
+2. Create conda environment
+
+    ```
+    cd cnv-patissier
+    # Create conda environment
+    conda create -n cnv-patissier python=3.6 anaconda
+    # Update python
+    conda update python
+    # Enable the cona environment
+    source activate cnv-patissier
+    ```
+
+3. Install requirements
+
+    ```
+    pip install -r requirements.txt
+    ```
+
+## Setup and usage
+
+### Overview of steps taken by CNV-patissier
+
+- CNV-patissier is run per capture and scans the gene sample sheets and runs all CNV callers on every sample per gene. 
+    - The output from each caller is created in `cnv-patissier/output/<capture>/<date-time-of-run>/<cnv-caller>/`
+    - The known CNV status and called CNVs are saved in a sqlite database in `cnv-patissier/output/` 
+    - A successful run settings file is written to `cnv-patissier/successful-run-settings/<capture>/<cnv-caller>/<gene>.toml`
+    - Logs are written to `cnv-patissier/logs/`
+- Each caller checks if there as been a successful run for that gene, if there has an no settings have changed (i.e. sample paths) then it moves onto the next. 
```

Comment:
Just a couple of typo's on this line:

Should this be:
"Each caller checks if there has been a successful run for that gene, if there has and no settings have changed (i.e. sample paths) then it moves onto the next. "

Reply:
thanks maria! corrected.